src/online2bin/online2-tcp-nnet3-decode-faster_word_timestamps.cc

```diff
@@ -426,6 +431,9 @@ int main(int argc, char *argv[]) {
               istringstream iss(msg);
               vector<string> message_vector{istream_iterator<string>{iss}, istream_iterator<string>{}};
               std::string message = "";
+              std::string block_word_start = "";
+              std::string block_word_end = "";
+              bool assigned = false;
```

Comment:
Rename this variable so its name says what is assigned here (so a reader don't have to find its usage in the code to find out the variable purpose).